projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-filtering.service.ts

```diff
@@ -42,6 +43,7 @@ export class IgxPivotFilteringService extends IgxFilteringService {
         this.prepare_filtering_expression(filteringTree, fieldName, term, conditionOrExpressionsTree, ignoreCase, fieldFilterIndex);
         dim.filter = filteringTree;
         grid.filteringPipeTrigger++;
+        grid.filterStrategy = new PivotFilteringStrategy();
```

Comment:
In case the user has set a custom filter strategy, for example:

```
<igx-pivot-grid  [filterStrategy]="noopFilterStrategy" ...>
```

This will probably overwrite it. 

We should not overwrite it if the user has set a different one.